server/database/sql/create_functions.sql

```diff
@@ -5,3 +5,103 @@ begin
 return NEW;
 end;
 $$ language plpgsql;
+
+CREATE OR REPLACE FUNCTION public.trigger_insert_words()
+    RETURNS trigger
+    LANGUAGE 'plpgsql'
+    COST 100
+    VOLATILE NOT LEAKPROOF
+AS $BODY$
+begin
+    insert into word(word, tracker) select
+		unnest(
+			array_remove(
+				regexp_split_to_array(
+					NEW.overview,
```

Comment:
@ChrisShep98 this is a function that takes the overview the user is posting and breaks it apart into words, then inserts each of those words into the word table with a foreign key that links it back to the tracker table